packages/scorekeeper-status-ui/src/App.tsx

```diff
@@ -50,7 +50,7 @@ const App = () => {
   const [jobs, setJobs] = useState([]);
   const [isLoading, setIsLoading] = useState(false);
   const [hasError, setHasError] = useState(false);
-  const [refreshInterval, setRefreshInterval] = useState(500); // Default refresh interval
+  const [refreshInterval, setRefreshInterval] = useState(5000); // Default refresh interval
```

Comment:
can you make this configurable instead hard coded ?

Reply:
Made constant